ci/debian.sh

```diff
@@ -36,6 +36,6 @@ apt install -y --no-install-recommends \
 	pkg-config \
 	xsltproc
 
-if [ "$BACKPORT_REPO" ]; then
+if [ -n "$BACKPORT_REPO" ]; then
```

Comment:
Again, this really works on all common shell being used on Linux distros. Being pedant like this does make sense on autotools configure script which is used on really different shells and OS, but here it's really not required.

Reply:
You are correct that it is not required, but most readers will be surprised by a behavior such as
```bash
var=0
if [ "$var" ]; then
    echo "This will still print"
fi
```

because that's where shell syntax deviates from expectations of many programming languages. Adhering to principle of least astonishment is a good practice.

Reply:
I'm sorry, but this project has more important issues than discussing shell style.

Reply:
Understandable.